assets/js/src/actions.js

```diff
@@ -0,0 +1,62 @@
+import { addAction } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+import { NAMESPACE, ACTION_PREFIX } from './constants';
+import {
+	trackEvent,
+	getProductFieldObject,
+	getProductImpressionObject,
+} from './utils';
+
+const trackListProducts = ( {
+	products,
+	listName = __( 'Product List', 'woo-gutenberg-products-block' ),
+} ) => {
+	trackEvent( 'view_item_list', {
+		event_category: 'engagement',
+		event_label: __( 'Viewing products', 'woo-gutenberg-products-block' ),
+		items: products.map( ( product, index ) => ( {
+			...getProductImpressionObject( product, listName ),
+			list_position: index + 1,
+		} ) ),
+	} );
+};
+
+const trackAddToCart = ( product, quantity = 1 ) => {
```

Comment:
The destructuring assignment was missed.

```suggestion
const trackAddToCart = ( { product, quantity = 1 } ) => {
```

Reply:
Thanks! Added here https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/261/commits/890cee55cbab9d497427ce0422788ebf20f2cc86